tesseract-server/src/handlers/logic_layer/geoadjacents.rs

```diff
@@ -108,24 +104,20 @@ pub fn logic_layer_geoadjacent(
         None => return Ok(HttpResponse::NotFound().json("Unable to access cube cache".to_string()))
     };
 
-    let cuts_map = match agg_query.cuts{
+    let cuts_map = match agg_query.cuts {
         Some(cut_map) => cut_map,
-        None => HashMap::new(),
+        None => return Ok(HttpResponse::NotFound().json("Malformed cuts".to_string())),
```

Comment:
This will be `None` when no cuts are provided, so we should specify that instead of saying they are malformatted.

Reply:
I tried raising an error over her but it seems that when we don't provide the cuts value it returns `Some({})` instead of `None`. I have solved this issue by putting a check in the get_relations method